### PR TITLE
Django 1.8

### PR DESCRIPTION
**Pull request by cbowman0**
Add Django 1.8 to the tests and other relevant sections.

As discussion in issue #1967.

### CODE REVIEW

**Comment by deniszh** [comment]
But does it work with Django 1.8 at all btw?

This should be ported to 1.0.x branch then.

**Comment by cbowman0** [comment]
I have found no issues in my testing​ so far.  I want try try a few more items before merging.

File: requirements.txt [hunk 1]
```diff
@@ -37,7 +37,7 @@
 #   deactivate
 #
 
-Django>=1.9
+Django>=1.8
```
Comment (DanCech) [inline requirements.txt]: Can we add `,<1.11.99` here to match setup.py?

Comment (cbowman0) [inline requirements.txt]: Yes.  Done.

**Comment by cbowman0** [comment]
I've found no issues.  Does anyone have anything specific to check?

**Comment by deniszh** [comment]
I think we can merge it.
Not sure do we need to backport to 1.0.x and release 1.0.2 or release 1.1.0 instead?
Maybe 1.0.2 is better, master is still very fresh.

**Comment by DanCech** [comment]
Yeah, we still need to do more testing on the refactored clustering before I'd consider it stable enough to cut 1.1.0

**Comment by cbowman0** [comment]
I submitted PR #1975 for the 1.0.x branch.  OK to merge here?

**Comment by DanCech** [comment]
Please do

**Pull request merged**